packages/ntcore-ts-client/src/lib/socket/socket.ts

```diff
@@ -168,7 +228,10 @@ export class NetworkTablesSocket {
   reinstantiate(serverUrl: string) {
     this.close();
     this.serverUrl = serverUrl;
-    this._websocket = new WebSocket(this.serverUrl, 'networktables.first.wpi.edu');
+    this._websocket = new WebSocket(this.serverUrl, [
+      'v4.1.networktables.first.wpi.edu',
+      'networktables.first.wpi.edu',
+    ]);
```

Comment:
This should probably be abstracted to a function? Particularly with the protocol array, it gets recreated several times throughout this file and will likely become problematic in the future when v4.2 inevitably comes out.